Taskfile.yml

```
@@ -131,6 +131,11 @@ tasks:
       - sh: 'test -f bin/go-sumtype{{exeExt}} || test -f bin/golangci-lint{{exeExt}}'
         msg: 'Please run `task init` first'
 
+  docs-lint:
+    desc: 'Run markdown linters'
```

```suggestion
  docs:
    desc: 'Run Markdown formatter and linter'
```

(to make it clear that files are changed by that command, not just checked)

---

Yes, but `--fix` should be useful (assuming it actually works :))